src/lbaf/Execution/lbsInformAndTransferAlgorithm.py

```diff
@@ -315,8 +308,8 @@ def execute(self, phase: Phase, distributions: dict, statistics: dict, a_min_max
             n_ignored, n_transfers, n_rejects = self.transfer_stage()
             n_proposed = n_transfers + n_rejects
             if n_proposed:
-                self.__logger.info(f"{n_proposed} proposed transfers, {n_transfers} occurred, {n_rejects} rejected "
-                                   f"({100. * n_rejects / n_proposed:.4}%)")
+                self.__logger.info(
+                    f"{n_proposed} proposed transfers, {n_transfers} occurred, {n_rejects} rejected ({100. * n_rejects / n_proposed:.4}%)")
```

Comment:
Is this a **requirement** as per language standard? 

I find splitting strings inelegant.

Reply:
It is usually depending on company standard 79-120 chars. 120 is very common and it fits 1280x.. resolution

Reply:
Yes but is this a **requirement**? I am really curious because oftentimes I realize that there are some obscure Python rules that people use, but that are not necessarily defined clearly anywhere.

Reply:
Philippe if by requirement you mean something in the language or interpreter, then it is not. This is the best practice developed by years of working with the language by companies. I agree with that. I am not saying anything about rules defining how many variables can be there in a function, or how nested can you go with ifs and for loops. As I mention many times there are many usefull tools that have these rules integrated.

Reply:
I see your point but on the other hand I resist it because it seems more subjective than logically strong. In fact it makes us depend on some specific IDE implementations, or even to obscure rules that seem to come out of nowhere, even when we think (oftentimes for good reason) that **our** way of doing it actually reads better.

My general take on these things is that **if it is allowed syntactically by the language then it is correct**. This gives more freedom to the developer to express his intent when writing the code. But I realize there is a trade-off to be made here between the developer's viewpoint, and the general community's view on these things.

Reply:
> I see your point but on the other hand I resist it because it seems more subjective than logically strong. In fact it makes us depend on some specific IDE implementations, or even to obscure rules that seem to come out of nowhere, even when we think (oftentimes for good reason) that **our** way of doing it actually reads better.
> 
> My general take on these things is that **if it is allowed syntactically by the language then it is correct**. This gives more freedom to the developer to express his intent when writing the code. But I realize there is a trade-off to be made here between the developer's viewpoint, and the general community's view on these things.

I disagree with that. IDE has nothing to do with language rules. It is just the tool, that makes code more readable and help catching obvious mistakes. 

Regarding `if it is allowed syntactically by the language then it is correct` I disagree as well. If there are no rules/standards/best practices, then code goes momentally not readable, non maintaiable. Smart people and companies developed that for years, so if something is good and helps I will take it. Freedom is very important, but devs need to cooperate and write code which is understable for others. If I would be hit by a car today, then tomorrow you need somebody, who can understand what's going on.

Reply:
They might be smart -- but I am smarter.

Reply:
Alright, I have re-split the print line (with the small difference that you will see @marcinwrobel1986 that I have made **explicit concatenation** by adding the `+` sign). I find implicit concatenation dangerous because if part of the concatenated string is replaced by a variable string, as could happen when doing a global replacement, this would result in a syntax error (as implicit concatenation cannot be done for a string that is evaluated at runtime), e.g.:
```
>>> a="123" "456"
>>> print(a)
123456
```
but
```
>>> b="456"
>>> a = "123" b
  File "<stdin>", line 1
    a = "123" b
              ^
SyntaxError: invalid syntax
```

Reply:
This will not happend, might if you do that on purpose. Always use f-string for a variable inside strings:
This is wrong `b="456"`, should be `b = "456"`
```Python3
b = "456"
a = f"123{b}"
```